content/tutorials/manage-the-rippled-server/troubleshooting/understanding-log-messages.md

```diff
@@ -60,12 +82,23 @@ Check the `[insight]` stanza in your `rippled`'s config file and confirm that yo
 This error has no other impact on the `rippled` server, which should continue to operate as normal except for the sending of StatsD metrics.
 
 
+## Check for upgrade
+
+The following message indicates that the server has detected that it is running an older software version than at least 60% of its trusted validators:
+
+```text
+LedgerMaster:ERR Check for upgrade: A majority of trusted validators are running a newer version.
```

Comment:
I wasn't able to reproduce this error personally so I had to "synthesize" it based on [the source code](https://github.com/ripple/rippled/pull/3447/files#diff-a7b89eed73c070e3b601e2d4de8bf4baR1106-R1111). Please check it for accuracy.

Reply:
Is `m_journal.error()` or `cerr` producing "LedgerMaster:ERR" ? I'm only uncertain if the string "ERR" is actually included.

Reply:
I think it's part of `m_journal.error()`. That's typically the case for similar log messages, though I don't have confirmation that this one works the normal way.

Reply:
I reran a test, the log line is:
```2020-Jul-13 14:20:40.799129030 UTC LedgerMaster:ERR Check for upgrade: A majority of trusted validators are running a newer version.```

Reply:
Very strictly speaking, we only count (and can only understand the format of the encoded version number of) rippled validators, as [Ed suggested](https://github.com/ripple/rippled/pull/3447#discussion_r445107811). But I don't think you need to change the explanation befow.

Reply:
FYI, the message "Check for upgrade: A majority of trusted validators are running a newer version." is also printed to the console.